webapp/sources/ldap-inventory/pom.xml

```diff
@@ -50,6 +50,13 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
     inventory, and one which managed provisioning.
   </description>
 
+  <licenses>
+    <license>
+      <name>GNU General Public License version 3 (GPLv3)</name>
```

Comment:
https://maven.apache.org/pom.html#Licenses says

> Using an SPDX identifier as the license name is recommended.

So it would be `GPL-3.0-or-later`

Reply:
We already try to use these identifiers in code comments and in other places (Rust's Cargo.toml, etc.)

Reply:
they don't do that in the example :/